create-release.py

```diff
@@ -281,10 +281,10 @@ def update_version_to_dev() -> None:
         sed(_source("README.md"), rf"elyra:{new_version} ", "elyra:dev ")
         sed(_source("etc/docker/kubeflow/README.md"), rf"kf-notebook:{new_version}", "kf-notebook:dev")
         # this does not goes back to dev
-        # sed(source('README.md'), rf"/v[0-9].[0-9].[0-9]", "/v{dev_version}")
+        # sed(source('README.md'), rf"/v[0-9]\+\.[0-9]\+\.[0-9]", "/v{dev_version}")
```

Comment:
Same, although I think the reference to `source` is incorrect, as is last digit handling and f-string placement...
```suggestion
        # sed(_source('README.md'), r"/v[0-9]\+\.[0-9]\+\.[0-9]\+", f"/v{dev_version}")
```

Reply:
removed per initial comment